dask/dataframe/core.py

```diff
@@ -1956,7 +1958,8 @@ def rolling(self, window, min_periods=None, center=False, win_type=None, axis=0)
         win_type : string, default None
             Provide a window type. The recognized window types are identical
             to pandas.
-        axis : int, default 0
+        axis : int | str| None, default 0
+            This parameter is deprecated with pandas>=2.1.
```

Comment:
Minor formatting nit

```suggestion
        axis : int, str, None, default 0
            This parameter is deprecated with ``pandas>=2.1``.
```